src/js/structs/JobRun.js

```diff
@@ -2,7 +2,7 @@ import DateUtil from '../utils/DateUtil';
 import Item from './Item';
 import JobTaskList from './JobTaskList';
 
-module.exports = class JobActiveRun extends Item {
+class JobRun extends Item {
   getDateCreated() {
```

Comment:
At present this only works for runs that belong to the `activeRuns`. The runs returned as part of the history have slightly dissimilar fields (`completedAt` vs `finishedAt` and maybe others...)